### ISSUE TITLE
[Adaptive Cards - Schema Explorer] - Screen Reader is conveying the incorrect role as 'Link' for 'View' button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8359]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Version 22H2 (OS Build 25300.1000)
URL: https://adaptivecards.io/explorer/
Browser: Edge Dev Version 111.0.1661.36 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Turn on the Narrator using 'ctrl+win+enter' key.
2. Open the above URL in edge dev browser.
3. Navigate to the 'View' button under 'Publish Adaptive Card schema' and observe the issue.

**Actual Result:**
Incorrect role as 'Link' is defined for the 'View' button. It is announcing as 'View' Link.

**Expected Result:**
Correct role should be defined for the 'View' option and screen reader should convey the correct role information as 'View' button.

**User Impact:**
It will impact the screen reader user if the incorrect role as link is announced by the screen reader as user will not able to understand the actual functionality of the element.

**MAS Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

![MAS4 1 2-Incorrect role as link is defined for the view button](https://user-images.githubusercontent.com/95913748/224327061-fc06d585-cac5-4156-97f1-833e0153c6b6.png)

https://user-images.githubusercontent.com/95913748/224330457-59cefc40-afe3-41d1-a9d4-9c45902eb794.mp4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

As this issue is still repro, so could you please let us know is there any future plan to fix this issue?
